test/integration/connect/envoy/case-terminating-gateway-simple/vars.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+export REQUIRED_SERVICES="s1 s1-sidecar-proxy s2 terminating-gateway-primary"
```

Comment:
Might be worth adding a comment here, or somewhere in the test, that excluding `s2-sidecar-proxy` is a very intentional and important decision for this test.

Reply:
Good call, done.